requirements.txt

```diff
@@ -8,10 +8,7 @@ ivadomed
 matplotlib
 nibabel
 numpy
-# onnxruntime>=1.5.1 requires `brew install libomp` on macOS.
-# So, pin to 1.4.0 to avoid having to ask users to install libomp.
-# ivadomed==2.5.0 would also do this, but #3035 is preventing that.
-onnxruntime==1.4.0
```

Comment:
It looks like moving away from `onnxruntime==1.4.0` has the side-effect of... _downgrading_ us from `ivadomed=2.7.4` to `ivadomed=2.4.0`? 

Digging deeper...

Reply:
Ahh, okay. The reason for this is partially due to https://github.com/ivadomed/ivadomed/issues/1042. We can't access the newer versions of `ivadomed` due to a dependency conflict re: `tensorboard`.

Reply:
`ivadomed==2.9.2` has been released, so I'm going to clean up my temporary commits a little, and then these issues should resolve themselves. (Famous last words...)

Reply:
Thank goodness!

![Screenshot from 2022-01-18 15-40-32](https://user-images.githubusercontent.com/16181459/150015266-6d65e63c-5403-49a0-b087-9d7b654064aa.png)

> `ivadomed-2.9.2`
> `tensorboard-1.15.0 tensorflow-1.15.5`
> `onnxruntime-1.7.0` 

Everything is installed in harmony together with no test failures. :heart: